bindata/bootkube/bootstrap-manifests/kube-apiserver-pod.yaml

```
@@ -138,3 +138,4 @@ spec:
   - hostPath:
       path: /var/log/kube-apiserver
     name: audit-dir
+
```

YAMLlint (1.37.1)

[warning] too many blank lines (1 > 0)

(empty-lines)